Documentation/anti-patterns.md

```diff
@@ -1,35 +1,35 @@
 ## Anti-Patterns
 
 ### Dialing in gRPC
-`grpc.Dial` is a function in the gRPC library that creates a virtual connection
+[`grpc.Dial`](https://pkg.go.dev/google.golang.org/grpc#Dial) is a function in the gRPC library that creates a virtual connection
 from the gRPC client to the gRPC server.  It takes a target URI (which can
 represent the name of a logical backend service and could resolve to multiple
-actual addresses) and a list of options, and returns a `ClientConn` object that
+actual addresses) and a list of options, and returns a [`ClientConn`](https://pkg.go.dev/google.golang.org/grpc#ClientConn) object that
 represents the connection to the server. The `ClientConn` contains one or more
 actual connections to real server backends and attempts to keep these
 connections healthy by automatically reconnecting to them when they break.
 
 The `Dial` function can also be configured with various options to
 customize the behavior of the client connection. For example, developers could
-use options such a `WithTransportCredentials` to configure the transport
+use options such a [`WithTransportCredentials`](https://pkg.go.dev/google.golang.org/grpc#WithTransportCredentials) to configure the transport
 credentials to use.
 
 While `Dial` is commonly referred to as a "dialing" function, it doesn't
-actually perform the low-level network dialing operation like `net.Dial` would.
+actually perform the low-level network dialing operation like [`net.Dial`](https://pkg.go.dev/net#Dial) would.
 Instead, it creates a virtual connection from the gRPC
 client to the gRPC server.
 
 `Dial` does initiate the process of connecting to the server, but it uses
 the ClientConn object to manage and maintain that connection over time. This is
 why errors encountered during the initial connection are no different from those
 that occur later on, and why it's important to handle errors from RPCs rather
-than relying on options like `FailOnNonTempDialError`, `WithBlock`, and
-`WithReturnConnectionError`.
+than relying on options like [`FailOnNonTempDialError`](https://pkg.go.dev/google.golang.org/grpc#FailOnNonTempDialError), [`WithBlock`](https://pkg.go.dev/google.golang.org/grpc#WithBlock), and
+[`WithReturnConnectionError`](https://pkg.go.dev/google.golang.org/grpc#WithReturnConnectionError).
 In fact, `Dial` does not always establish a connection to servers by default. The connection
 behavior is determined by the load balancing policy being used. For instance, an "active"
 load balancing policy such as Round Robin attempts to maintain a constant connection, while the default "pick first"
-policy delays connection until an RPC is executed. To explicitly initiate a connection, you can employ
-the WithBlock option.
+policy delays connection until an RPC is executed.Instead of using the WithBlock option, which may not be recommended 
```

Comment:
Space after period, please.  Also, please make another pass to rewrap the lines at 80 columns.